README.md

```diff
@@ -1,107 +1,115 @@
+- **Add-ons**: Heroku apps have a rich inventory of [Heroku Add-ons](https://elements.heroku.com/addons) that provide tools and services for developing, extending, and operating your app.
 
 ## Reference Functions Framework
 
 ### Overview
-The Reference Functions Framework is an example of how to lift-and-shift Salesforce Functions to run securely as 
-Heroku Apps in [Private Spaces](https://devcenter.heroku.com/articles/private-spaces).
 
-**The Reference Functions Framework is as-is and not supported by Salesforce or Heroku.**  You may fork the repository 
-and make customizations as needed.
+The Reference Functions Framework is an example of how to migrate Salesforce Functions to securely run as Heroku apps in [Private Spaces](https://devcenter.heroku.com/articles/private-spaces).
+
+**The Reference Functions Framework is provided as-is and is not supported by Salesforce or Heroku.** 
+You can fork the repository and make customizations as needed.
 
 The Reference Functions Framework includes:
-- **Connected Apps** for authentication (trust boundary) and authorization (determines functions' Organization access).
+
+- **Connected Apps** for authentication and authorization to create the trust boundary and determine the functions’ Salesforce org access.
 - **Apex Classes** and an **Apex Trigger** as APIs to invoke functions, supporting both synchronous and asynchronous invocation.
-- A new **Custom Metadata Type** (`FunctionReference__mdt`) representing deployed, invocable functions.
-- A new **Custom Object** (`AsyncFunctionInvocationRequest__c`) that tracks asynchronous requests and handles asynchronous responses.
-- **Language-specific proxies** (`Java` and `Node`) are deployed alongside and in front of a function to validate, enrich, and manage function requests.
+- A new **Custom Metadata Type**, `FunctionReference__mdt`, that represents deployed, invocable functions.
+- A new **Custom Object**, `AsyncFunctionInvocationRequest__c`, that tracks asynchronous requests and handles asynchronous responses.
+- **Language-specific proxies**, `Java` and `Node`, that are deployed with functions to validate, enrich, and manage function requests.
+
+Together, these elements allow you to deploy your existing Salesforce Functions as Heroku apps with minimal disruption to your org.
 
 ### Architecture
-The following diagram shows how Reference Functions Framework components interact with functions deployed as Heroku Apps.
-- **Orange** boxes indicated Reference Functions Framework components provided in this repository.  
-- **Green** boxes indicate customer set up or developed components that implement Reference Functions Framework APIs and function code. 
+
+The following diagram shows how Reference Functions Framework components interact with functions deployed as Heroku apps.
+
+- **Orange** boxes indicate Reference Functions Framework components provided in this repository.
```

Comment:
Please change to **Yellow**.  I changed the diagram's color and didn't change here.  Thank you.